src/tests/ftest/container/full_pool_container_create.py

```diff
@@ -32,24 +27,24 @@ def test_no_space_cont_create(self):
         self.prepare_pool()
 
         # query the pool
-        self.d_log.debug("querying pool info")
-        self.pool.get_info()
-        self.d_log.debug("queried pool info")
+        self.log.info("Pool Query before write")
+        self.pool.set_query_data()
+        self.log.info(
+            "Pool %s query data: %s\n", self.pool.uuid, self.pool.query_data)
 
         # create a container
         try:
-            self.d_log.debug("creating container")
-            self.cont = DaosContainer(self.context)
-            self.cont.create(self.pool.pool.handle)
-            self.d_log.debug("created container")
+            self.log.info("creating container 1")
+            cont = DaosContainer(self.context)
+            cont.create(self.pool.pool.handle)
+            self.log.info("created container 1")
         except DaosApiError as excep:
-            self.d_log.error("caught exception creating container: "
-                             "{0}".format(excep))
+            self.log.error("caught exception creating container: "
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions